src/Character/index.tsx

```diff
@@ -2,14 +2,14 @@ import React, { useEffect, useState } from 'react';
 
 import Panel, { FlipPanel } from '../Panel';
 import defaultSound from '../assets/flip.mp3';
-// @ts-expect-error the minified file is not in the type declarations
-import { Howl } from 'howler/dist/howler.min.js';
```

Comment:
Is this just a local conflict or are you also proposing to remove the sound playback feature?

Reply:
added Howler back in -- accidental removal.